h2o-bindings/bin/custom/python/gen_maxrglm.py

```diff
@@ -1,4 +1,44 @@
 def class_extensions():
+    def coef(self):
```

Comment:
I think one thing that users might see is that MaxRGLM creates multiple models while GLM makes only one. So they will get a list of coefficients for each model. If they build a pipeline, they will need to be aware of this and know which model/index to select

Reply:
@neemah2o This is correct and they should expect that because they are building multiple models here.  However, the indices are pretty generic.  If they are looking for best one predictor model, it will be index = 0, for best two predictors model, index=1, and so on and so forth.  I will check and make sure the documentation mentions that.

Reply:
I think I will add an index allowing them to choose the coefficients of a particular size.  Thanks for the feedback.